en/admin/scripts.rst

```diff
@@ -0,0 +1,113 @@
+
+:meta-keywords: scripts
+:meta-description: Various CUBRID scripts support CUBRID Managements
+
+***************
+cubrid scripts
+***************
+
+.. _cubrid_scripts_unloaddb_sh:
+
+unloaddb.sh script
+==================
+
+To shorten the execution time of **unloaddb** of a large size database, it is possible to run **cubrid unloaddb** processes concurrently over groups of different tables.
+**unloaddb.sh** is a shell script that automates this process.
+
+To evenly distribute unloaddb load over multiple child processes, the **unloadb.sh** gets the size information of the tables to be unloaded from the system catalog (this requires dba privileges).
+
+Tables to be unloaded are allocated to each child process based on the table size, and each child process simultaneously unloads the allocated tables.
+
+
+.. warning::
+
+	**restrictions:**
+	  * Require Linux Environments
+	  * Only users with **DBA privileges** can run this script
+
+The following is [options] used in **unloaddb.sh**.
+
+.. program:: sh unloaddb.sh
+
+.. option:: -u user
```

Comment:
At warning section, you wrote that this shell script is executed by only dba.
So, I think that the -u option don't need.
How about your think ?

Reply:
Users belonging to the DBA group can also execute this script, even they are not 'dba' user.

Reply:
OK, I got it.
Then, It seems that it should be marked as a user with dba privileges.

Reply:
In the warning section, "Only users with **DBA privileges** can run this script", isn't it enough?

Reply:
OK.

Reply:
You haven't approved it, can I proceed to merge?